determined-ai/determined
# feat: update profiler charts

## Description

WEB-662





## Test Plan
Need to enable chart flag `?f_chart=on` in url
- Check if profiler charts look the same as before switching the chart component
- Check if data is correct




## Commentary (optional)





## Checklist

  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.

## Ticket
